### PR TITLE
Upgrade dependencies to target Android Q

### PR DESCRIPTION
**Pull request by nighthawk24**
Set compile and target SDK version to 29, Migrate android support dependencies to androidx, update corresponding layouts. Stringify resources in layouts. Improve code formatting and remove empty spaces.

### CODE REVIEW

**Review by zquestz — APPROVED** [review]

File: app/src/main/res/layout/tor_one.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Neutrino Wallet does an excellent job of preventing your peers from learning of your incoming payments. However, outgoing spends are still sent sent in the clear meaning your remote peers could potentially associate your spends with your IP address. This tutorial will show you how to set up Orbot to enhance the privacy of your spends."
```
Comment (hieronymus777) [inline app/src/main/res/layout/tor_one.xml:27]: Nit: typo in this string `...are still sent sent in the...` should be `...are still sent in the...`

Comment (hieronymus777) [reply app/src/main/res/layout/tor_one.xml:27]: Nevermind :D I see it's fixed in the strings.xml file

**Review by hieronymus777 — COMMENTED** [review]
LGTM

**Review by zquestz — APPROVED** [review]
LGTM. Thanks!

**Comment by cpacia** [comment]
Thanks for putting this together! Only change I'm going to request is fix this regression in both the send and receive activities where the amount is entered. 

Notice the $ (and ₿ symbol when toggled) is not aligned properly and it's showing the error help text with a 0 above.

Screenshot from 2019-08-01 21-28-29

**Review by cpacia — CHANGES_REQUESTED** [review]
Thanks for putting this together! Only change I'm going to request is fix this regression in both the send and receive activities where the amount is entered.

Notice the $ (and ₿ symbol when toggled) is not aligned properly and it's showing the error help text with a 0 above.

Screenshot from 2019-08-01 21-28-29

**Comment by zquestz** [comment]
@nighthawk24 any chance you can look into the regression mentioned by @cpacia ?

**Comment by nighthawk24** [comment]
Thank you @cpacia & @zquestz for reviewing and testing the large change. I wanted to get the base setup right before working on feature requests. I've handled the alignment in the layout for now, though the layouts can be refactored to constraint layouts in the future.

**Comment by zquestz** [comment]
@cpacia this nearly ready? Would be nice to land this.

**Pull request merged**